### DISCUSSION TITLE
Searching for branch feature has been removed on creating pull request page

### DISCUSSION THREAD

**Discussion by NinaHwang** [community/community#129256] [Pull Requests]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

I believe there was a search bar but it disappeared..
<img width="424" alt="image" src="https://github.com/community/community/assets/79563565/5eda2bd4-5494-45ce-8693-99fa0ff3a83d">

Update:
UI seems weird here too.
<img width="371" alt="image" src="https://github.com/community/community/assets/79563565/44f33ca0-2be0-4dcd-b6f5-7dcc352ee3a5">

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by the-spanish-guy** [comment]
There was a search bar but it also disappeared for me, I believe they will fix it soon if they didn't actually remove it

**Reply by digismack** [reply]
Hopefully they didn't remove it. Searching here is a core part of the PR workflow and is already firmly lodged into my muscle-memory. 😄

**Comment by hirocaster** [comment]
I hope it's a bug and has temporarily disappeared.

**Comment by CescaMerlin** [comment]
I'm experiencing the same problem

**Comment by Ganthology** [comment]
Having the same issue, it slows down the team's workflow. Github please bring it back 😢

**Comment by ghost** [comment]
Is there any GIthub release note mentioning this?

**Comment by jasonhai-bee** [comment]
I believe this issue just happened this early morning.
We also faced this issue when tried to create a pull request for my team. At first, I though it was an issue with my laptop, but then I realized we can not filter the branch for any organization in my GitHub.

Is there anyone from the GitHub team can help to look into it and revert the changes please!

**Comment by MinSomai** [comment]
as an alternative you can install github CLI

https://cli.github.com/

and use this simple command to create PR:

```bash
 gh pr create -B main -H MY-BRANCH-TO-MERGE
```

**Comment by Ankit-Kumar-13** [comment]
I hope it's a bug , Not a feature !! 🫤

**Comment by chiragvidani** [comment]
I am also surprised to see this change. 

Can anyone from GitHub please acknowledge and fix this?

**Comment by agungkes** [comment]
At first I thought it was my browser at fault, but it's not.
I hope this is not feature

**Comment by nilmond010** [comment]
Why has this been removed? A bug Thums down👎 I hope it's not a feature

**Comment by dextercampos** [comment]
Same issue. I thought it was my browser blocking some scripts or something.

**Comment by g0rr0g** [comment]
Thanks for opening this issue @NinaHwang Yes, this is a super irritating "feature" for our team at the moment. Please fix it :pray:

**Comment by jayfaaa** [comment]
alternative : 

[repolink]/compare/[Base]...[compare]

example 
https://github.com/my_company/my_repo/compare/my_base...my_compare

**Reply by chiragvidani** [reply]
agree, that is an alternative, till we have the fix.

**Comment by Moistbobo** [comment]
microsoft please i pay for windows and github pro dont do this to me

**Comment by Muralidevan27** [comment]
I'm not sure why they removed search feature , it's very crucial, also the listing of branches doesn't have any sorting mechanism, hard to follow , please bring it back

**Comment by nilmond010** [comment]
Just got an update from the GitHub support team, this is a bug

**Comment by menshawii** [comment]
A workaround for now is using the browser url.
Example:
```
https://github.com/{repo}/compare/{baseBranch}...{compareBranch}
```

**Comment by dextercampos** [comment]
Temporary Fix:

1. Install a browser extension that'll run/append a javascript to a specific website. I used [Custom Site JS extension](https://chromewebstore.google.com/detail/custom-site-js/pclbdoihplpfbkmebiaojnhfbgdgkcol?hl=en)
2. Add the following script as seen in the screenshot:
```
var githubPrFixBranchFilter = function () {
    var selectMenuFilterElements = document.getElementsByClassName('SelectMenu-filter');
    if (selectMenuFilterElements.length === 2) {
        selectMenuFilterElements[0].parentElement.parentElement
            .insertBefore(selectMenuFilterElements[0], selectMenuFilterElements[0].parentElement);
        selectMenuFilterElements[1].parentElement.parentElement
            .insertBefore(selectMenuFilterElements[1], selectMenuFilterElements[1].parentElement);
    }
};
githubPrFixBranchFilter();
window.navigation.addEventListener('navigate', (event) => {
    githubPrFixBranchFilter();
});
```
![Screenshot 2024-06-21 at 4 20 00 PM](https://github.com/community/community/assets/27075066/a51e31e5-64ec-471e-9806-a9b06c302df7)

I hope this helps you temporarily while waiting for Github to fix it 😄

**Reply by shubham-nanostuffs** [reply]
Good solution..!, working fine

**Reply by ardianpramana** [reply]
Thank you!

**Reply by Admin7727** [reply]
terima kasih..

**Reply by Admin7727** [reply]
June 26 : updated line >> 'SelectMenu-filter' >> 'SelectMenu-list'

**Comment by cuongcpt** [comment]
Premium for this feature LOL

**Reply by vishveshjain** [reply]
❤️

**Comment by lifo-linxuanli** [comment]
please bring it back.

**Comment by 0verflowme** [comment]
do I have to pay for this feature or what?

**Comment by RaW-Git** [comment]
Probably they can't fix this anymore, since they can't find the correct base branch for the fix PR 😅

**Comment by MinSomai** [comment]
its back 🥳

**Comment by AkhiljithSoftnotions** [comment]
Its back guys

**Comment by cheeselemon** [comment]
My life has never been worse when I found out that this stopped working. 

Now, my life is happy =)

**Comment by ghostinhershell** [comment]
Hello @NinaHwang and all, thanks so much for reporting this issue. This issue is now resolved. Please feel free to follow-up here and let us know if you're still running into this bug on your end. Cheers!
